mautic/mautic
# [UX] Differentiate types of emails

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

This PR adds the differences between email types.
Based on:
https://docs.mautic.org/en/5.x/channels/emails.html
https://joeykeller.com/knowledge-base-2/mautic-onboarding/create-your-first-segment-or-template-email-with-mautic/

![image](https://github.com/mautic/mautic/assets/116097999/9e8126d8-894a-4c37-b67d-0cf38d2219c8)




#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Channels > Emails
3. Create new
4. See the comparison
